[CI] UpgradeClusterClientYamlTestSuiteIT test {p0=mixed_cluster/100_analytics_usage/Basic test for usage stats on analytics indices} failing

**Build Scans:**
- [elasticsearch-intake #10124 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/xdxhgudeyzqfe)
- [elasticsearch-pull-request #32754 / 8.16.0_bwc-snapshots](https://gradle-enterprise.elastic.co/s/oeubqjos6z35k)

**Reproduction Line:**
```
./gradlew ":x-pack:qa:rolling-upgrade:v8.16.0#oneThirdUpgradedTest" -Dtests.class="org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=mixed_cluster/100_analytics_usage/Basic test for usage stats on analytics indices}" -Dtests.seed=1F8EC5D2A10E5838 -Dtests.bwc=true -Dtests.locale=ses-ML -Dtests.timezone=US/Hawaii -Druntime.java=22
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Bp0%3Dmixed_cluster%2F100_analytics_usage%2FBasic%20test%20for%20usage%20stats%20on%20analytics%20indices%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [mixed_cluster/100_analytics_usage:58]: expected [2xx] status code but api [xpack.usage] returned [500 Internal Server Error] [{"error":{"root_cause":[{"type":"transport_serialization_exception","reason":"Failed to deserialize response from handler [ContextRestoreResponseHandler[ContextRestoreResponseHandler[UnregisterChildTransportResponseHandler[unregisterChildNode=releaseOnce[org.elasticsearch.tasks.TaskManager$$Lambda/0x00007f4260532480@7fbc7fec], handler=[org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener/org.elasticsearch.action.support.ContextPreservingActionListener/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.tasks.TaskManager$1{org.elasticsearch.rest.action.RestCancellableNodeClient$1@2280056a}{CancellableTask{Task{id=2415, type='transport', action='cluster:monitor/xpack/usage', description='', parentTask=unset, startTime=1727214431758, headers={}, startTimeNanos=2717149897642}, reason='null', isCancelled=false}}/org.elasticsearch.action.support.TransportAction$$Lambda/0x00007f42603e4fd8@5d66c6a6/org.elasticsearch.xpack.security.action.filter.SecurityActionFilter$$Lambda/0x00007f42603ee5f8@413991f9/org.elasticsearch.action.support.master.TransportMasterNodeAction$$Lambda/0x00007f42603e6f60@8f0bbf6], action=cluster:monitor/xpack/usage, childRequest=TermOverridingMasterNodeRequest[newMasterTerm=4 in org.elasticsearch.protocol.xpack.XPackUsageRequest/R0YSH8cwSJiob6817CJOnA:2415], childConnection=NodeChannels[{v8.16.0-1}{MBlV9Y4aRmWod2wLqj871w}{HacxLJNDT8CR_Oxyk8vxjQ}{v8.16.0-1}{127.0.0.1}{127.0.0.1:40147}{cdfhilmrstw}{8.16.0}{7000099-8515000}], taskManager=org.elasticsearch.tasks.TaskManager@47987ab2]]]]","stack_trace":"org.elasticsearch.transport.TransportSerializationException: Failed to deserialize response from handler [ContextRestoreResponseHandler[ContextRestoreResponseHandler[UnregisterChildTransportResponseHandler[unregisterChildNode=releaseOnce[org.elasticsearch.tasks.TaskManager$$Lambda/0x00007f4260532480@7fbc7fec], handler=[org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener/org.elasticsearch.action.support.ContextPreservingActionListener/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.tasks.TaskManager$1{org.elasticsearch.rest.action.RestCancellableNodeClient$1@2280056a}{CancellableTask{Task{id=2415, type='transport', action='cluster:monitor/xpack/usage', description='', parentTask=unset, startTime=1727214431758, headers={}, startTimeNanos=2717149897642}, reason='null', isCancelled=false}}/org.elasticsearch.action.support.TransportAction$$Lambda/0x00007f42603e4fd8@5d66c6a6/org.elasticsearch.xpack.security.action.filter.SecurityActionFilter$$Lambda/0x00007f42603ee5f8@413991f9/org.elasticsearch.action.support.master.TransportMasterNodeAction$$Lambda/0x00007f42603e6f60@8f0bbf6], action=cluster:monitor/xpack/usage, childRequest=TermOverridingMasterNodeRequest[newMasterTerm=4 in org.elasticsearch.protocol.xpack.XPackUsageRequest/R0YSH8cwSJiob6817CJOnA:2415], childConnection=NodeChannels[{v8.16.0-1}{MBlV9Y4aRmWod2wLqj871w}{HacxLJNDT8CR_Oxyk8vxjQ}{v8.16.0-1}{127.0.0.1}{127.0.0.1:40147}{cdfhilmrstw}{8.16.0}{7000099-8515000}], taskManager=org.elasticsearch.tasks.TaskManager@47987ab2]]]]\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:422)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler$2.doRun(InboundHandler.java:390)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:992)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1570)\nCaused by: java.io.IOException: Can't read unknown type [111]\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericValue(StreamInput.java:887)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readMap(StreamInput.java:741)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readMap(StreamInput.java:726)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericValue(StreamInput.java:862)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericMap(StreamInput.java:789)\n\tat org.elasticsearch.xcore@9.0.0-SNAPSHOT/org.elasticsearch.xpack.core.ml.MachineLearningFeatureSetUsage.<init>(MachineLearningFeatureSetUsage.java:70)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteableCollectionAsList(NamedWriteableAwareStreamInput.java:46)\n\tat org.elasticsearch.xcore@9.0.0-SNAPSHOT/org.elasticsearch.xpack.core.action.XPackUsageResponse.<init>(XPackUsageResponse.java:27)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:64)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:26)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1785)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1754)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1489)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1476)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1489)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1476)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:419)\n\t... 6 more\n"}],"type":"transport_serialization_exception","reason":"Failed to deserialize response from handler [ContextRestoreResponseHandler[ContextRestoreResponseHandler[UnregisterChildTransportResponseHandler[unregisterChildNode=releaseOnce[org.elasticsearch.tasks.TaskManager$$Lambda/0x00007f4260532480@7fbc7fec], handler=[org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener/org.elasticsearch.action.support.ContextPreservingActionListener/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.tasks.TaskManager$1{org.elasticsearch.rest.action.RestCancellableNodeClient$1@2280056a}{CancellableTask{Task{id=2415, type='transport', action='cluster:monitor/xpack/usage', description='', parentTask=unset, startTime=1727214431758, headers={}, startTimeNanos=2717149897642}, reason='null', isCancelled=false}}/org.elasticsearch.action.support.TransportAction$$Lambda/0x00007f42603e4fd8@5d66c6a6/org.elasticsearch.xpack.security.action.filter.SecurityActionFilter$$Lambda/0x00007f42603ee5f8@413991f9/org.elasticsearch.action.support.master.TransportMasterNodeAction$$Lambda/0x00007f42603e6f60@8f0bbf6], action=cluster:monitor/xpack/usage, childRequest=TermOverridingMasterNodeRequest[newMasterTerm=4 in org.elasticsearch.protocol.xpack.XPackUsageRequest/R0YSH8cwSJiob6817CJOnA:2415], childConnection=NodeChannels[{v8.16.0-1}{MBlV9Y4aRmWod2wLqj871w}{HacxLJNDT8CR_Oxyk8vxjQ}{v8.16.0-1}{127.0.0.1}{127.0.0.1:40147}{cdfhilmrstw}{8.16.0}{7000099-8515000}], taskManager=org.elasticsearch.tasks.TaskManager@47987ab2]]]]","caused_by":{"type":"i_o_exception","reason":"Can't read unknown type [111]","stack_trace":"java.io.IOException: Can't read unknown type [111]\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericValue(StreamInput.java:887)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readMap(StreamInput.java:741)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readMap(StreamInput.java:726)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericValue(StreamInput.java:862)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericMap(StreamInput.java:789)\n\tat org.elasticsearch.xcore@9.0.0-SNAPSHOT/org.elasticsearch.xpack.core.ml.MachineLearningFeatureSetUsage.<init>(MachineLearningFeatureSetUsage.java:70)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteableCollectionAsList(NamedWriteableAwareStreamInput.java:46)\n\tat org.elasticsearch.xcore@9.0.0-SNAPSHOT/org.elasticsearch.xpack.core.action.XPackUsageResponse.<init>(XPackUsageResponse.java:27)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:64)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:26)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1785)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1754)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1489)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1476)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1489)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1476)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:419)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler$2.doRun(InboundHandler.java:390)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:992)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1570)\n"},"stack_trace":"org.elasticsearch.transport.TransportSerializationException: Failed to deserialize response from handler [ContextRestoreResponseHandler[ContextRestoreResponseHandler[UnregisterChildTransportResponseHandler[unregisterChildNode=releaseOnce[org.elasticsearch.tasks.TaskManager$$Lambda/0x00007f4260532480@7fbc7fec], handler=[org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener/org.elasticsearch.action.support.ContextPreservingActionListener/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener/org.elasticsearch.tasks.TaskManager$1{org.elasticsearch.rest.action.RestCancellableNodeClient$1@2280056a}{CancellableTask{Task{id=2415, type='transport', action='cluster:monitor/xpack/usage', description='', parentTask=unset, startTime=1727214431758, headers={}, startTimeNanos=2717149897642}, reason='null', isCancelled=false}}/org.elasticsearch.action.support.TransportAction$$Lambda/0x00007f42603e4fd8@5d66c6a6/org.elasticsearch.xpack.security.action.filter.SecurityActionFilter$$Lambda/0x00007f42603ee5f8@413991f9/org.elasticsearch.action.support.master.TransportMasterNodeAction$$Lambda/0x00007f42603e6f60@8f0bbf6], action=cluster:monitor/xpack/usage, childRequest=TermOverridingMasterNodeRequest[newMasterTerm=4 in org.elasticsearch.protocol.xpack.XPackUsageRequest/R0YSH8cwSJiob6817CJOnA:2415], childConnection=NodeChannels[{v8.16.0-1}{MBlV9Y4aRmWod2wLqj871w}{HacxLJNDT8CR_Oxyk8vxjQ}{v8.16.0-1}{127.0.0.1}{127.0.0.1:40147}{cdfhilmrstw}{8.16.0}{7000099-8515000}], taskManager=org.elasticsearch.tasks.TaskManager@47987ab2]]]]\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:422)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler$2.doRun(InboundHandler.java:390)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:992)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:27)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.base/java.lang.Thread.run(Thread.java:1570)\nCaused by: java.io.IOException: Can't read unknown type [111]\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericValue(StreamInput.java:887)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readMap(StreamInput.java:741)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readMap(StreamInput.java:726)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericValue(StreamInput.java:862)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.StreamInput.readGenericMap(StreamInput.java:789)\n\tat org.elasticsearch.xcore@9.0.0-SNAPSHOT/org.elasticsearch.xpack.core.ml.MachineLearningFeatureSetUsage.<init>(MachineLearningFeatureSetUsage.java:70)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.common.io.stream.NamedWriteableAwareStreamInput.readNamedWriteableCollectionAsList(NamedWriteableAwareStreamInput.java:46)\n\tat org.elasticsearch.xcore@9.0.0-SNAPSHOT/org.elasticsearch.xpack.core.action.XPackUsageResponse.<init>(XPackUsageResponse.java:27)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:64)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.action.ActionListenerResponseHandler.read(ActionListenerResponseHandler.java:26)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1785)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.read(TransportService.java:1754)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1489)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1476)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1489)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.read(TransportService.java:1476)\n\tat org.elasticsearch.server@9.0.0-SNAPSHOT/org.elasticsearch.transport.InboundHandler.doHandleResponse(InboundHandler.java:419)\n\t... 6 more\n"},"status":500}]
```

**Issue Reasons:**
- [main] 2 consecutive failures in step 8.16.0_bwc-snapshots
- [main] 2 failures in test test {p0=mixed_cluster/100_analytics_usage/Basic test for usage stats on analytics indices} (0.4% fail rate in 471 executions)
- [main] 2 failures in step 8.16.0_bwc-snapshots (2.4% fail rate in 84 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -281,12 +281,6 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.MlJobIT
   method: testOutOfOrderData
   issue: https://github.com/elastic/elasticsearch/issues/113477
-- class: org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT
-  method: test {p0=mixed_cluster/100_analytics_usage/Basic test for usage stats on analytics indices}
-  issue: https://github.com/elastic/elasticsearch/issues/113497
-- class: org.elasticsearch.xpack.esql.qa.mixed.EsqlClientYamlIT
-  method: test {p0=esql/60_usage/Basic ESQL usage output (telemetry)}
-  issue: https://github.com/elastic/elasticsearch/issues/113502
 - class: org.elasticsearch.xpack.ml.integration.MlJobIT
   method: testCreateJobsWithIndexNameOption
   issue: https://github.com/elastic/elasticsearch/issues/113528

